config/console.ini

```diff
@@ -4,16 +4,40 @@
 
 ### CONSOLE OPTIONS ###
 
+## EnableLogRotation (bool)
+# True to enable log rotation. See the below settings for more control over log rotation.
+# True implies logging to files. Filenames will be auto-generated and stored in LogFileDirectory.
+EnableLogRotation = false
+
 ## LogOut (string)
-# Filename to log normal output
+# Filename to log normal output when log rotation is disabled
 # Use - for console output
 LogOut = -
 
 ## LogErr (string)
-# Filename to log errors
+# Filename to log errors when log rotation is disabled
 # Use - for console output
 LogErr = error.log
```

Comment:
I'm assuming this suggestion is based on your comment above? It sounds like you're suggesting a lot of what I was thinking regarding log providers, but like I said, I'd rather get into that when I add structured logging in a future PR.

I definitely agree that having `LogOut`, `LogErr`, and `EnableLogRotation` is confusing. I'll see if I can consolidate these settings so that they're not necessary. I might need to wait until I add a system for log providers though. Alternatively, I might abandon these changes and do it all at the same time, because the more I think about it there's really no point in introducing this log rotation method and _then_ moving it all into log providers...I still need to keep thinking on that